Environment_Engine/Modify/Transform.cs

```diff
@@ -47,7 +48,7 @@ public static Space Transform(this Space space, TransformMatrix transform, doubl
                 return null;
             }
 
-            Space result = space.GetShallowClone() as Space;
+            Space result = space.ShallowClone() as Space;
```

Comment:
```suggestion
            Space result = space.ShallowClone();
```

Reply:
Is that better @alelom - will let you decide as you have the current spatial awareness of the situation.

Reply:
Yes sorry I realized you could do that only later